docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -361,6 +361,85 @@
         in bare metal we can have jobs that could effect other jobs because
         they are running in the same process space.</para>
       </sect3>
+
+      <sect3 id="YAML_Thor_and_hThor_Memory">
+        <title>Thor and hThor Memory </title>
+
+        <para>The "memory" sections were introduced initially for Thor only,
```

Comment:
Is this information necessary to share? I don't think anyone needs to know it was Thor only at first.